topicquests/qs-demo
# Changed mainlayout

Changed the mainlayout to be used if person is logged in or not. This currently changes app to only need one layout. 
How to test:
1. Check for signup and sign in buttons are viewable.
2. Reduce screen size to ensure sign up and sign in buttons hide and menu icon displays right side
3. Check that sign up button and sign in button are viewable when clicking right nav. menu
4. Register new user if do not have existing one.
5. Log in.
6. After successful login should see logout button.
7. Do steps 2 and three but this time should react with logout button
